### PR TITLE
chore: fix lint issues for IAM resources

### PR DESCRIPTION
**Pull request by ShiChangkuo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

* this is the **last** PR to fix lint issues for IAM service.

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
$ make testacc TEST="./huaweicloud/services/acceptance/iam" TESTARGS="-run TestAccIdentitACL"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/iam -v -run TestAccIdentitACL -timeout 360m -parallel 4
=== RUN   TestAccIdentitACL_basic
=== PAUSE TestAccIdentitACL_basic
=== RUN   TestAccIdentitACL_apiAccess
=== PAUSE TestAccIdentitACL_apiAccess
=== CONT  TestAccIdentitACL_basic
=== CONT  TestAccIdentitACL_apiAccess
--- PASS: TestAccIdentitACL_basic (11.27s)
--- PASS: TestAccIdentitACL_apiAccess (11.28s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/iam       11.318s

$ make testacc TEST="./huaweicloud/services/acceptance/iam" TESTARGS="-run TestAccIdentityAccessKey"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/iam -v -run TestAccIdentityAccessKey -timeout 360m -parallel 4
=== RUN   TestAccIdentityAccessKey_basic
=== PAUSE TestAccIdentityAccessKey_basic
=== RUN   TestAccIdentityAccessKey_secret
=== PAUSE TestAccIdentityAccessKey_secret
=== CONT  TestAccIdentityAccessKey_basic
=== CONT  TestAccIdentityAccessKey_secret
--- PASS: TestAccIdentityAccessKey_secret (6.99s)
--- PASS: TestAccIdentityAccessKey_basic (11.37s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/iam       11.412s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_acl_test.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ func getIdentitACLResourceFunc(c *config.Config, state *terraform.ResourceState)
 		}
 		if len(v.AllowAddressNetmasks) == 0 && len(v.AllowIPRanges) == 1 &&
 			v.AllowIPRanges[0].IPRange == "0.0.0.0-255.255.255.255" {
-			return nil, fmtp.Errorf("Identity ACL for console access <%s> not exists", state.Primary.ID)
+			return nil, fmt.Errorf("Identity ACL for console access <%s> not exists", state.Primary.ID)
```
Comment (houpeng80) [inline huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_acl_test.go]:
```suggestion
			return nil, fmt.Errorf("identity ACL for console access <%s> not exists", state.Primary.ID)
```

File: huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_acl_test.go [hunk 2]
```diff
@@ -37,20 +37,20 @@ func getIdentitACLResourceFunc(c *config.Config, state *terraform.ResourceState)
...
```
Comment (houpeng80) [inline huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_acl_test.go]:
```suggestion
			return nil, fmt.Errorf("identity ACL for console access <%s> not exists", state.Primary.ID)
```

File: huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_access_key_test.go [hunk 3]
```diff
@@ -26,7 +25,7 @@ func getIdentityAccessKeyResourceFunc(c *config.Config, state *terraform.Resourc
 	}
 
 	if found.AccessKey != state.Primary.ID {
-		return nil, fmtp.Errorf("Access Key not found")
+		return nil, fmt.Errorf("Access Key not found")
```
Comment (houpeng80) [inline huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_access_key_test.go:28]:
```suggestion
		return nil, fmt.Errorf("access Key not found")
```

Comment (ShiChangkuo) [reply huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_access_key_test.go:28]: the **Access Key** maybe more suitable as it is a technical terms.

File: huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_acl_test.go (hunk 1 above)
Comment (ShiChangkuo) [reply huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_acl_test.go]: fixed

File: huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_acl_test.go (hunk 2 above)
Comment (ShiChangkuo) [reply huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_acl_test.go]: fixed

**Pull request merged**